### ISSUE TITLE
[0.9.3 staging-1968] House value doesn't update properly.

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#21208]
- [ ] 1. House value is not update when you unclaim/transfer property.
Step to reproduce:
- player 1 build some room and add some furniture, have 11,5 house value:
![image](https://user-images.githubusercontent.com/45708377/113123071-e8f45b80-921c-11eb-87f3-71b0d445ddb5.png)
- unclaim property:
![image](https://user-images.githubusercontent.com/45708377/113123107-f1e52d00-921c-11eb-9e31-bf5169d3c943.png)
- submit:
![image](https://user-images.githubusercontent.com/45708377/113123151-fd385880-921c-11eb-8f2e-bf40d0f3fb8b.png)
- but I still have 11,5 house value, but this is UI problem
- take logging and chop some tree:
![image](https://user-images.githubusercontent.com/45708377/113123438-438db780-921d-11eb-8ea3-522b0f691010.png)

- [ ] 2. House value is not upgrade properly when you leave resident.
Step to reproduce:
- player 1 build some room and add some furniture, have 11,5 house value:
![image](https://user-images.githubusercontent.com/45708377/113123679-75068300-921d-11eb-8404-1c3f7ef8bc20.png)
- leave residency:
![image](https://user-images.githubusercontent.com/45708377/113123718-7fc11800-921d-11eb-93ea-cb7f2fd1f50b.png)
- I have 5,77 now
- if another player become a resident< then my UI will show me 11,5 again, but I don't have this bonus:
![image](https://user-images.githubusercontent.com/45708377/113124587-55bc2580-921e-11eb-9cac-8435d477be14.png)

**Comment by SlayksWood** [comment]
Didn't check but looks like regression

**Comment by abaza121** [comment]
I don't think changing the ownership should change residency, deleting deed or unclaiming however should change it so I will investigate that.

**Comment by abaza121** [comment]
https://github.com/StrangeLoopGames/Eco/pull/8115 this issue seems to be fixed with this.

**Comment by SlayksWood** [comment]
#20061

**Comment by Daniel-Magulado** [comment]
Reconfirming.
Tested on 0.9.4 Develop-286.

Unclaiming a property with an existing house value that affects the player retains the house value. However, it does not affect the experience multiplier despite visually compelling so.
Please watch the video below:

https://user-images.githubusercontent.com/77248866/118428690-9b677b80-b702-11eb-90f3-2ac407bfa346.mp4

But on the other hand, leaving residency now works as intended.
I think the housing value depends on the residency changes. Unclaiming doesn't seem to trigger internal changes in residency that's why housing value doesn't get updated promptly after changes.

**Comment by Daniel-Magulado** [comment]
In addition, if you build a room and place furniture in it, and then claim the land after that, the housing value doesn't get updated until you pick one of the furniture and place it again.


https://user-images.githubusercontent.com/77248866/118431755-a1ad2600-b709-11eb-9af8-2e56b3314ccf.mp4

Do note that there's no deed existing yet here after completing your room and placing furniture. The last step is to claim the land.

 Basically, build a room, place furniture then claim the land.

**Comment by abaza121** [comment]
I'm addressing the issues on the comments and the unclaim via edit on the next PR

**Comment by TwixterZ** [comment]
Tested issue on 9.4 Staging 2022, issue with house value update when un-claim and re-claim land is fixed and housing value will update accordingly. 

However, I have found new issue in regards to housing value update : 
![image.png](https://images.zenhubusercontent.com/6017bedcbf0e6bfb14e12003/244e297b-65dd-4097-b90b-d5882af4e051)

What have happen is : 
- Created a fresh world 
- Skipped all tutorial
- I have construct a house with housing values as per tooltip above
- Restart tutorial 
- Complete housing tutorial OR skip all tutorial again 
- Housing value UI will show as pictured above

**Comment by TwixterZ** [comment]
Fix tested and confirmed as of 9.4 Staging-2071
